cms/djangoapps/contentstore/views/tests/test_transcript_settings.py

```diff
@@ -422,7 +422,7 @@ def test_transcript_upload_handler_with_invalid_transcript(self):
         """
         Tests the transcript upload handler with an invalid transcript file.
         """
-        transcript_file_stream = BytesIO('An invalid transcript SubRip file content')
+        transcript_file_stream = BytesIO(b'An invalid transcript SubRip file content')
```

Comment:
Question: The change above switched from using `BytesIO` to `StringIO`, while the change here changed the string to be a bytestring.  Is there a reason for fixing the above differently from the way it was fixed here?

Reply:
Good call, had fixed these during different points of the process so didn't notice that. I'll update.  This to also be stringIO.